include/lcp-catlist.php

```diff
@@ -117,7 +117,9 @@ private function lcp_categories(){
     if ( is_array($this->lcp_category_id) ){
       return array('category__and' => $this->lcp_category_id);
     } else {
-      if($this->utils->lcp_not_empty('child_categories') && ($this->params['child_categories'] === ('no' || 'false') )){
+      if($this->utils->lcp_not_empty('child_categories') && 
+         (($this->params['child_categories'] === 'no' )
```

Comment:
Sorry, you are right. I copied the code from another WordPress install of mine into the Git repository I used for uploading. Guess something got lost when copy and pasting into the Git repo ;)